### PR TITLE
WIP complete fade stop animation in route

### PR DESCRIPTION
**Pull request by chris-saburo**
Done it.

### CODE REVIEW

File: src/components/BaseNavigationBottom.vue [hunk 1]
```diff
@@ -6,25 +6,40 @@
...
+              >
                 <HomeIcon class="m-auto" />
                 Home
               </router-link>
             </li>
             <li class="text-white inline-block m-2 mb-1 text-xs">
-              <router-link class="text-white router-link" :to="{ name: 'Stats' }">
+              <router-link
+                :class="activeItem == 'stats' ? 'text-white router-active-link' : 'text-white router-normal-link'"
+                class="text-white router-link"
```
Comment (atharva3010) [inline src/components/BaseNavigationBottom.vue:20]: text-white is a duplicate class

File: src/components/BaseNavigationBottom.vue [hunk 2]
```diff
@@ -6,25 +6,40 @@
...
+              >
                 <StatsIcon class="m-auto" />
                 Stats
               </router-link>
             </li>
             <li class="text-white inline-block m-2 mb-1 text-xs">
-              <router-link class="text-white router-link" :to="{ name: 'JumpStartLists' }">
+              <router-link
+                :class="activeItem == 'bluedelta' ? 'text-white router-active-link' : 'text-white router-normal-link'"
+                class="text-white router-link"
```
Comment (atharva3010) [inline src/components/BaseNavigationBottom.vue:30]: here too

File: src/components/BaseNavigationBottom.vue [hunk 3]
```diff
@@ -6,25 +6,40 @@
...
+              >
                 <MemoStatslIcon class="m-auto" />
                 Blue Delta
               </router-link>
             </li>
             <li class="text-white inline-block m-2 mb-1 text-xs">
-              <router-link class="text-white router-link" :to="{ name: 'ArticleLists' }">
+              <router-link
+                :class="activeItem == 'article' ? 'text-white router-active-link' : 'text-white router-normal-link'"
+                class="text-white router-link"
```
Comment (atharva3010) [inline src/components/BaseNavigationBottom.vue:40]: here too

File: src/App.vue [hunk 4]
```diff
@@ -1,9 +1,9 @@
 <template>
   <div id="app" class="text-center">
-    <transition name="fade" mode="out-in">
+    <transition name="slide" mode="out-in">
```
Comment (atharva3010) [inline src/App.vue:3]: I don't think there is a class for slide animation

Comment (chris-saburo) [reply src/App.vue:3]: Done it.

File: src/components/BaseNavigationBottom.vue (hunk 3 above)
Comment (chris-saburo) [reply src/components/BaseNavigationBottom.vue:40]: Done it.

File: src/components/BaseNavigationBottom.vue (hunk 1 above)
Comment (chris-saburo) [reply src/components/BaseNavigationBottom.vue:20]: Done it.

**Comment by atharva3010** [comment]
@mort3za can you review the active class/selected button logic here?

File: src/components/BaseNavigationBottom.vue [hunk 5]
```diff
@@ -6,25 +6,40 @@
...
+                :to="{ name: 'Dashboard' }"
+              >
                 <HomeIcon class="m-auto" />
                 Home
               </router-link>
             </li>
             <li class="text-white inline-block m-2 mb-1 text-xs">
-              <router-link class="text-white router-link" :to="{ name: 'Stats' }">
+              <router-link
+                :class="activeItem == 'stats' ? 'router-active-link' : 'router-normal-link'"
```
Comment (mort3za) [inline src/components/BaseNavigationBottom.vue:19]: This is an automatic process in vue-router. You don't need to handle it yourself. Check this API reference please https://router.vuejs.org/api/#active-class

**Pull request closed without merging**